### PR TITLE
feat: Support tf2e-v10 protocol in go-svrquery

### PR DESCRIPTION
**Pull request by nikbanerjee-unity**

### CODE REVIEW

**Comment by nikbanerjee-unity** [comment]
## Test Evidence

```
❯ ./go-svrquery -proto tf2e-v10 -key=Y0hIS1ZIS0FoREVNSWZ6Cg== -addr 107.6.103.22:37016
{
	"Prefix": -1,
	"Command": 78,
	"Version": 10,
	"HealthFlags": {
		"None": true,
		"PacketLossIn": false,
		"PacketLossOut": false,
		"PacketChokedIn": false,
		"PacketChokedOut": false,
		"SlowServerFrames": false,
		"Hitching": false,
		"DOS": false
	},
	"BuildName": "R5pc_r5dev_J7887_CL3980367_2023_02_15_13_24",
	"Datacenter": "central us",
	"GameMode": "survival",
	"Port": 37015,
	"Platform": "Crossplay",
	"PlaylistVersion": "",
	"PlaylistNum": 14756,
	"PlaylistName": "dev_default",
	"NumClients": 0,
	"NumBotClients": 0,
	"MaxClients": 65,
	"TotalClientsConnectedEver": 0,
	"Map": "mp_lobby",
	"PlatformPlayers": {
		"PC_ORIGIN": 0,
		"PC_STEAM": 0,
		"PS4": 0,
		"PS5": 0,
		"SWITCH": 0,
		"X1": 0,
		"XB5": 0
	},
	"AverageFrameTime": 0,
	"MaxFrameTime": 0,
	"AverageUserCommandTime": 0,
	"MaxUserCommandTime": 0,
	"CommitMemory": 3991,
	"ResidentMemory": 1268,
	"MaxRounds": 0,
	"RoundsWonIMC": 0,
	"RoundsWonMilitia": 0,
	"TimeLimit": 0,
	"MaxScore": 0,
	"Phase": 0,
	"TimePassed": 0,
	"CurrentEntityPropertyCount": 1452,
	"MaxEntityPropertyCount": 250000,
	"Teams": null,
	"Clients": null
}
```

File: lib/svrquery/protocol/titanfall/query.go [hunk 1]
```diff
@@ -174,7 +174,11 @@ func (q *queryer) Query() (resp protocol.Responser, err error) {
 	}
 
 	if i.Version > 2 {
-		if i.Version >= 9 {
+		if i.Version >= 10 {
```
Comment (stevenh) [inline lib/svrquery/protocol/titanfall/query.go:177]: Is it worth converting this to a switch statement now the cases are increasing?

**Review by stevenh — APPROVED** [review]
One minor comment, if you feel like changing it, otherwise good to go.

File: lib/svrquery/protocol/titanfall/query.go (hunk 1 above)
Comment (nikbanerjee-unity) [reply lib/svrquery/protocol/titanfall/query.go:177]: I do agree it's worth switching over, but I'm going to opt to leave it this time as I've gone through the testing cycle and want to get this delivered shortly. If I had longer I'd look to refactor it a little in general including your recommendation, but I will leave that for when I pick up v11 in the future and start with the refactor if that's ok? 🙂

**Review by aserra54 — APPROVED** [review]

**Review by lwaddicor — APPROVED** [review]
Looks fine to me

**Pull request merged**